tests/assets/repository/index.yaml

```diff
@@ -0,0 +1,60 @@
+{
+  "generated": "2020-05-06T20:30:58.581950",
+  "operator": {
+    "honeybee-radiance": [
+      {
+        "name": "honeybee-radiance",
+        "version": "1.2.3",
+        "appVersion": "2.3.2",
+        "keywords": [
+          "honeybee",
+          "radiance",
+          "ladybug-tools"
+        ],
+        "maintainers": [
+          {
+            "name": "mostapha",
+            "email": "mostapha@ladybug.tools"
+          }
+        ],
+        "home": "https://ladybug.tools/honeybee-radiance/docs",
+        "sources": [
+          "https://github.com/ladybug-tools/honeybee-radiance"
+        ],
+        "icon": "https://www.ladybug.tools/assets/img/logo.png",
+        "deprecated": null,
+        "description": "An operator to run all things honeybee and radiance!",
+        "url": "operators/honeybee-radiance-1.2.3.tgz",
+        "created": "2020-05-06T20:30:58.576675",
+        "digest": "e8215c80cede7eed5d5447634661ed01fc15485fd5374d3ec574456eb96ae289"
+      }
+    ]
+  },
+  "recipe": {
+    "daylight-factor": [
+      {
+        "name": "daylight-factor",
+        "version": "0.0.1",
+        "keywords": [
+          "ladybug-tools",
+          "radiance"
+        ],
+        "maintainers": [
+          {
+            "name": "mostapha",
+            "email": "mostapha@ladybug.tools"
+          }
+        ],
+        "home": null,
+        "sources": null,
+        "icon": "https://www.ladybug.tools/assets/img/logo.png",
+        "deprecated": null,
+        "description": "An awesome workflow to run daylight factor!",
+        "license": null,
+        "url": "recipes/daylight-factor-0.0.1.tgz",
+        "created": "2020-05-06T20:30:58.581874",
+        "digest": "dfe964ce20bca8a1685b52cfce2b810d10cc31fb9ac15daa1128f3d712228d48"
```

Comment:
interesting! how does one provide this?

Reply:
Automatically generated. The index is generated from commands by either reading a folder or packaging and adding a new operator/recipe to a repository folder.

Reply:
The idea here is to make sure that if someone overwrites a version somehow, any dependencies are aware of it and can decide to either accept the changes or have their recipe broken. It's WIP but a powerful concept.